Uplift/Networking/NetworkManager.swift

```diff
@@ -101,6 +101,21 @@ struct NetworkManager {
         apollo.fetch(query: AllGymNamesQuery()) { result, error in
             guard error == nil,
                 let gymNamesData = result?.data?.gyms else { return }
+
+            let gyms = gymNamesData.map({ GymNameId(name: $0?.name, id: $0?.id) })
+
+            completion(gyms)
+        }
+    }
+
+    func getGymNames(completion: @escaping ([GymNameId]) -> Void, failure: (() -> Void)? = nil) {
+        apollo.fetch(query: AllGymNamesQuery()) { result, error in
+            guard error == nil,
+                let gymNamesData = result?.data?.gyms else {
+                    failure?()
+                    return
+            }
+
```

Comment:
aren't these functions the same except this second one just has an optional failure argument?